modules/virt-deploying-ssp.adoc

```diff
@@ -3,10 +3,10 @@
 // * virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc
 
 :_content-type: PROCEDURE
-[id="virt-deploying-tto_{context}"]
-= Deploying the Tekton Tasks Operator resources
+[id="virt-deploying-ssp_{context}"]
+= Deploying the Scheduling, Scale, and Performance (SSP) resources
 
-The Tekton Tasks Operator (TTO) tasks and example pipelines are not deployed by default when you install {VirtProductName}. To deploy TTO resources, enable the `deployTektonTaskResources` feature gate in the `HyperConverged` custom resource (CR).
+The SSP operator tasks and example pipelines are not deployed by default when you install {VirtProductName}. To deploy SSP operator resources, enable the `deployTektonTaskResources` feature gate in the `HyperConverged` custom resource (CR).
```

Comment:
I'd write it as:

1. `The SSP operator example Tekton Tasks and Pipelines ...`
2. `To deploy SSP operator's Tekton resources, enable the `deployTektonTaskResources` ...`